presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixTableProperties.java

```diff
@@ -148,7 +149,10 @@ public PhoenixTableProperties()
         if (rowkeysCsv == null) {
             return Optional.empty();
         }
-        return Optional.of(Arrays.asList(StringUtils.split(rowkeysCsv, ',')));
+
+        List<String> rowkeys = Arrays.stream(StringUtils.split(rowkeysCsv, ',')).map(String::trim).collect(toList());
+
+        return Optional.of(rowkeys);
```

Comment:
```suggestion

        return Optional.of(Arrays.stream(StringUtils.split(rowkeysCsv, ','))
                .map(String::trim)
                .collect(toImmutableList()));
```

It would be better if we could in-line rowkeys

Reply:
Thanks @Praveen2112, I made the modifications and pushed new changes.